profiling/nvprof-focused-connector/kp_nvprof_focused_connector.cpp

```diff
@@ -117,7 +117,17 @@ void kokkosp_begin_parallel_reduce(const char* name, const uint32_t devID,
 void kokkosp_end_parallel_reduce(const uint64_t kID) {
   focusedConnectorExecuteEnd();
 }
+    
+void kokkosp_begin_fence(const uint64_t kID) {
+  *kID = nextKernelID++;
+  currentKernel = getFocusedConnectorInfo(name, FENCE);
+  focusedConnectorExecuteStart();
```

Comment:
this is not necessary. What this does is turning nvtx profiling on and off. But this is only a fence anyway, so likely no CUDA operation will happen